packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -947,58 +947,59 @@
                             data-ng-bind=":: ServiceContactCtrl.getStatusLabel()"
                         ></p>
                     </div>
+                </div>
+                <!-- End of UNIVERSAL DIRECTORY -->
 
-                    <div class="form-group mt-4">
```

Comment:
this part is moved outside of parent div which contains an "ng-if" on the country to be able to edit the form for every country